packages/test-utils/src/testInterface.ts

```diff
@@ -0,0 +1,10 @@
+interface HavingTestInterface { testInterfaceType?: any }
+
+/**
+ * Gets the Testing Interface
+ * @param obj The object to get the testing interface from
+ * @returns The testing interface
+ */
+export function getTI<T extends HavingTestInterface> (obj: T): NonNullable<T['testInterfaceType']> {
+    return (obj as unknown as any)['testInterface']!
```

Comment:
Could maybe rename the field to `_testInterface` (and the type to `_TestInterfaceType`) so that IDE/eslint know that it is ok to have the "unused local varible". 

(Eslint seems to be allow it also without underscore, but good to follow the convention anyway)

https://github.com/streamr-dev/network/blob/7eb22634afa250917420a432320484f2d65e4a3c/.eslintrc.js#L59

Reply:
I would not do this, because these variables are not unused in reality.

Reply:
Adding the prefix would communicate exactly that: i.e. that the variable is not unused even when IDE/`eslint` doesn't find any usages for it.